### PR TITLE
Journal: Don't explode on unpublished posts

### PR DESCRIPTION
**Pull request by zspencer**
See: https://github.com/zinc-collective/convene/issues/1026
See: https://github.com/zinc-collective/convene/issues/898

Welp, that's what I get for not testing that case! Now there's a test, it's not great... needs some actual assertions; but it exercised the error!

### CODE REVIEW

File: spec/furniture/journal/entries_request_spec.rb [hunk 1]
```diff
@@ -6,7 +6,17 @@
   let(:room) { journal.room }
   let(:member) { create(:person, spaces: [space]) }
 
-  describe "POST /entries" do
+  describe "#index" do
+    it "shows all the journal entries" do
+      sign_in(space, member)
+      entries = create_list(:journal_entry, 3, journal: journal)
```
Comment (anaulin) [inline spec/furniture/journal/entries_request_spec.rb:12]: Would be nice to explicitly make sure in the test that we have both published and un-published entries and are displaying all of them. But not blocking to merge this in.

**Review by anaulin — APPROVED** [review]
🐛🔨

File: spec/furniture/journal/entries_request_spec.rb (hunk 1 above)
Comment (zspencer) [reply spec/furniture/journal/entries_request_spec.rb:12]: Good call! I'll do that then merge!

Comment (anaulin) [reply spec/furniture/journal/entries_request_spec.rb]: Thank you for adding that! ✅

**Pull request merged**